proposed/container.md

```diff
@@ -81,6 +83,10 @@ into the same container (or another container) instead of the delegate container
 The interfaces and classes described as well as relevant exception are provided as part of the
 [psr/container](https://packagist.org/packages/psr/container) package. (still to-be-created)
 
+Packages providing a psr container implementation should declare that they provide psr/container-implementation 1.0.0
+
+Projects requiring an implementation should require psr/container-implementation 1.0.0
+
```

Comment:
First time I see that `psr/log` has a `psr/log-implementation`, and I guess it's the same for PSR-7. OK for this if it's common for PSRs.